tests/data/abs_files/orig/arxiv/papers/0704/0704.0615v1.abs

```diff
@@ -0,0 +1,22 @@
+------------------------------------------------------------------------------
+\\
+arXiv:0704.0615
+From: Trevor Bruen TB <tbruen@gmail.com>
```

Comment:
Please replace the authors email with something like <author@example.com> instead of having the authors email in the code.